docs/resources/Audit_Log.md

```diff
@@ -142,6 +142,7 @@ Whenever an admin action is performed on the API, an entry is added to the respe
 | channel_id                    | [invite](#DOCS_RESOURCES_INVITE/invite-object) or [guild scheduled event](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object) | snowflake                                                                   | channel for invite code  or guild scheduled event changed                                                                                                                         |
 | code                          | [invite](#DOCS_RESOURCES_INVITE/invite-object)                                                                              | string                                                                                        | invite code changed                                                                                                                                     |
 | color                         | [role](#DOCS_TOPICS_PERMISSIONS/role-object)                                                                                | integer                                                                                       | role color changed                                                                                                                                      |
+| communication_disabled_until | [user](#DOCS_RESOURCES_USER/user-object) | ?ISO8601 timestamp                              | the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) is set |
```

Comment:
1. let's specify non-nullable type `?ISO8601 timestamp`. I don't think `null` values are included as part of the [change](https://discord.com/developers/docs/resources/audit-log#audit-log-change-object-audit-log-change-structure)

2. would be good to keep description more consistent with surrounding description and maybe make the guild scope clear since we are just using 'user' -- maybe something like: *"user guild timeout state changed"* ? Also it's probably ok to not include the link to the support article here.

3. NIT: can we align the column boundaries `|` the best we can with surrounding rows to stay consistent :)

4. I know other member properties like `deaf` point to the user obj but I feel like all of these member properties, including `communication_disabled_until`, should prob link to the [member obj](https://discord.com/developers/docs/resources/guild#guild-member-object) where they are defined instead. @devsnek what do you think?